src/transformer/descriptor/nodeTypes/nodeTypes.ts

```diff
@@ -0,0 +1,14 @@
+import type * as ts from 'typescript';
+
+const NodeTypesGlobalFile: string = 'node_modules/@types/node/globals.d.ts';
+
+export function IsNodeType(node: ts.Node): boolean {
```

Comment:
Maybe it's better if we specify "nodejs" to be clear we're not meaning ts.Node

Reply:
@Pmyl I've made the changes. 

I've also updated the commit message to include the change in a version, this is a fix/feature. I would say fix. I don't think we want to mock any node js globals types ever...